Configuration/TypoScript/setup.typoscript

```diff
@@ -66,10 +66,19 @@ page {
     kitodo-pageView-searchInDocument = EXT:dlf/Resources/Public/JavaScript/PageView/SearchInDocument.js
     kitodo-pageView-pageView = EXT:dlf/Resources/Public/JavaScript/PageView/PageView.js
     kitodo-search-suggest = EXT:dlf/Resources/Public/JavaScript/Search/Suggester.js
-
   }
 }
 
+[typo3.branch=="11.5"]
+  page.inlineSettings {
+    dlfAssets = /typo3conf/ext/dlf/Resources/Public/
+  }
+[else]
+  page.inlineSettings {
+    dlfAssets = /_assets/007831ab53eafe3f6be2b9b1baf2a8d6/
```

Comment:
According to [documentation](https://docs.typo3.org/permalink/t3coreapi:Administration/Upgrade/MigrateToComposer/AssetMigration#migration) we can't rely on the asset folder hash being the same for every installation. Is there no better way to access the assets without the need to hard-code the hash here?